superset/db_engine_specs/trino.py

```diff
@@ -220,3 +221,12 @@ def update_params_from_encrypted_extra(
         except json.JSONDecodeError as ex:
             logger.error(ex, exc_info=True)
             raise ex
+
+    @classmethod
+    def get_dbapi_exception_mapping(cls) -> Dict[Type[Exception], Type[Exception]]:
+        # pylint: disable=import-error,import-outside-toplevel
+        from requests import exceptions as requests_exceptions
+
+        return {
+            requests_exceptions.ConnectionError: SupersetDBAPIDisconnectionError,
```

Comment:
There is a suggestion but it's not related to this PR. the `SupersetDBAPIDisconnectionError` might be an ambiguous naming here. the [ConnectionError](https://github.com/psf/requests/blob/1e62a3ec18e19f85ddae03b4cbbdf0b4c62834c0/requests/exceptions.py#L49) is a generic exception for connection-related issues rather than a specific error for *disconnection*, so I think we might change it to `SupersetDBAPIConnectionError`.

Reply:
Hehe, interesting that you raised that, because I thought the exact same thing when I came across that exception 😆 I'll see if renaming it is a good fit with the current places it's used, and if so I'll refactor accordingly.